testutils/file_utils.go

```diff
@@ -25,24 +25,41 @@ const osDarwin = "darwin"
 func BuildViamServer(tb testing.TB) string {
 	tb.Helper()
 
-	command := "server-static"
-	if runtime.GOOS == osDarwin {
-		command = "server"
-	}
+	buildOutputPath := tb.TempDir()
+	serverPath := filepath.Join(buildOutputPath, "viam-server")
 
-	builder := exec.Command("make", command)
+	var builder *exec.Cmd
+
+	if runtime.GOOS != "windows" {
+		command := "server-static"
+		if runtime.GOOS == osDarwin {
+			command = "server"
+		}
+		builder = exec.Command("make", command)
+		builder.Env = append(os.Environ(), "TESTBUILD_OUTPUT_PATH="+buildOutputPath)
+	} else {
+		// we don't have access to make on Windows, so copy the build command from the Makefile.
```

Comment:
best I can do, open to other ideas!

Reply:
Windows ships with the `winget` package manager now and it has a port of GNU make available. I'm experimenting w/ using that avoid duplicating this build logic in my rdk fork.